.tools/gen_config.sh

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+mkdir -p ~/.paddle
```

Comment:
Maybe instead of generating this file with a script (unit test breaks without running this script), you can check in this file into the code base, maybe under `testdata`:https://github.com/golang/go/blob/master/src/net/testdata/hosts . And add argument to override the default location `~/.paddle` to the test file.